jetty-ee10/jetty-ee10-osgi/jetty-ee10-osgi-boot/src/main/java/org/eclipse/jetty/ee10/osgi/annotations/AnnotationParser.java

```diff
@@ -151,7 +151,7 @@ public int compare(String o1, String o2)
             {
                 hasDotPath = true;
             }
-            else if (!token.endsWith(".jar") && !token.endsWith("/"))
+            else if (!FileID.isJavaArchvie(token) && !token.endsWith("/"))
```

Comment:
Typo: `isJavaArchvie`.

Reply:
Done (FYI, this was on code that is in a project that is not being compiled)